lib/service.rb

```diff
@@ -2,6 +2,12 @@
 # type, the configuration data, and the payload for the current call.
 class Service
   UTF8 = "UTF-8".freeze
+  DEPRECATION_NOTE = [
+    "**NOTE:** This service been marked for deprecation: https://developer.github.com/changes/2018-04-25-github-services-deprecation/",
```

Comment:
Just noticed this pull request—it might be worth updating this to "This service has been marked for deprecation ...". See https://github.com/github/github-services/pull/1248.